src/lib/zkapp.ts

```diff
@@ -178,6 +179,7 @@ function wrapMethod(
                 accountUpdates: [],
                 fetchMode: inProver() ? 'cached' : 'test',
                 isFinalRunOutsideCircuit: false,
+                numberOfRuns: Infinity,
```

Comment:
Why do we use `Infinity` inside a checked computation here?

Reply:
err.. I introduced this `numberOfRuns` field on the transaction because the transaction code is run twice, and I wanted to identify whether I was in the first one, so that I wouldn't display the same warning twice. However, then in prove / compile I had no good idea how to set it; I just wanted it to be more than 0. So I opted for `Infinity`. Maybe 2 would also be fine

Reply:
Maybe we can introduce a type to have the constraints that it can only be the value `1`, `2` or an ignore value. That way it's easier to tell what `numberOfRuns` should be at any point in time. Up to you though!